.env.prod

```diff
@@ -3,4 +3,4 @@ JOURNEYPLANNER_HOST=https://api.entur.io/journey-planner/v2
 GEOCODER_HOST=https://api.entur.io/geocoder/v1
 FIREBASE_CONFIG='{"apiKey":"AIzaSyDBzat6zxRmQQQQsEpI3YiZ9uX4V9d-Bgg","authDomain":"entur-tavla-prod.firebaseapp.com","databaseURL":"https://entur-tavla-prod.firebaseio.com","projectId":"entur-tavla-prod","storageBucket":"entur-tavla-prod.appspot.com","messagingSenderId":"1025677335075","appId":"1:1025677335075:web:7e5e1f495564c1a0dcef96","measurementId":"G-GJLJPD7LYB"}'
 MAPBOX_TOKEN=pk.eyJ1IjoiZW50dXIiLCJhIjoiY2tlbGQ1OHQzMG10dTM1azdvOXJvcXExeCJ9.s65sEHnbUfmuqRsDwjByxw
-MAPBOX_STYLE=mapbox://styles/entur/cj8kdr9ea4wva2rojj29tisp0
+MAPBOX_STYLE=mapbox://styles/entur/ckfi7v87704jn19o71b6z02bp
```

Comment:
Dette betyr vel at alle stoppestadene forsvinn i scooters-kartet på Kompakt visning? Er det noko me ønsker?

Reply:
Hmmm, tenkte ikke på det. Kanskje ha 2 kartstiler for nå?

Reply:
Ta 2 kartstiler enn så lenge da :)

Reply:
Det er det i PR'en nå 😃